lib/Driver/ToolChain.cpp

```diff
@@ -30,6 +30,7 @@
 
 using namespace swift;
 using namespace swift::driver;
+using namespace llvm;
```

Comment:
Ah, please *don't* do this. My comments last time were specific to SmallVector and SmallVectorImpl, two types used commonly enough that we import them into the `swift` namespace in swift/Basic/LLVM.h. That's not a fixed list (in general, it's fine to add more llvm/ADT/ types), but we don't want *all* LLVM things going unqualified.